Configurations/10-main.conf

```diff
@@ -1343,6 +1345,7 @@ my %targets = (
         thread_scheme    => "winthreads",
         dso_scheme       => "win32",
         apps_aux_src     => add("win32_init.c"),
+        build_scheme     => add("VC-common", { separator => undef }),
```

Comment:
Since this affects installation directories, I'm not at all sure about this.  I much prefer to have explicit flavors (for example `VC-W64` that you removed) than "trying to be smart".

Reply:
See above.